go.mod

```diff
@@ -3,8 +3,10 @@ module github.com/cilium/hubble
 go 1.14
 
 require (
-	github.com/cilium/cilium v1.8.0-rc1.0.20200525154943-1ea19605ae0c
+	github.com/cilium/cilium v1.8.0-rc1.0.20200706163532-1a5cc4a05511
```

Comment:
the `rc1` here is still confusing, it's just the latest tag on master.

The commit is correct

```
❯ g show 1a5cc4a05511
commit 1a5cc4a05 (HEAD -> master, origin/master, origin/HEAD)
Author: Paul Chaignon <paul@cilium.io>
Date:   9 hours ago
```